packages/teleterm/src/ui/ClusterLogout/useClusterLogout.ts

```diff
@@ -19,8 +17,8 @@ export function useClusterLogout({ clusterUri, onClose, clusterTitle }: Props) {
         await ctx.workspacesService.setActiveWorkspace(null);
       }
     }
-    ctx.workspacesService.removeWorkspace(clusterUri);
     ctx.connectionTracker.removeItemsBelongingToRootCluster(clusterUri);
+    ctx.workspacesService.removeWorkspace(clusterUri);
```

Comment:
I suppose we could bring the original order of these two calls to, couldn't we? So that `removeWorkspace` is called before `removeItemsBelongingToRootCluster` – if it's not necessary then there's no reason to change it. ;)

Reply:
Done